compiler/ballerina-parser/src/test/resources/types/tuple-type/tuple_type_source_10.bal

```diff
@@ -0,0 +1,9 @@
+type T1 [@annot {} int, @annot {} string];
+
+type T2 [int, @annot {} string];
+
+type T1 [int, @annot {} string...];
```

Comment:
Since this is a negative test with a syntax error, shall we add it separately?
(Under `// Valid source test` we only keep the correct syntax. Any test with syntax errors goes under `// Recovery test` in `TupleTypeTest`.)